src/elements/cta/package.json

```diff
@@ -17,6 +17,7 @@
     "react": "^16.7.0"
   },
   "dependencies": {
-    "@uswitch/trustyle.button-link": "^0.5.0"
+    "@uswitch/trustyle.button-link": "^0.5.0",
+    "lodash": "^4.17.15"
```

Comment:
please dont add lodash, it makes the kitten sad 

![](https://media.giphy.com/media/CM1rHbKDMH2BW/giphy.gif)

Reply:
lodash is still here